src/views/Home/components/LostAndFoundCard/index.tsx

```diff
@@ -178,14 +198,49 @@ const LostAndFoundCard = () => {
             </Button>
           </Grid>
         </Grid>
+        {foundReports && foundReports.length > 0 && (
+          <>
+            <Card>
+              <CardHeader
+                title={
+                  <Grid container alignItems="center" position="relative">
+                    {/* Badge positioned on the top-left corner */}
+                    <Badge
+                      badgeContent={foundReports.length}
+                      color="error"
+                      overlap="circular"
+                      showZero={false}
+                      sx={{
+                        position: 'absolute',
+                        top: '15px',
+                        left: '160px',
+                      }}
+                    />
+                    <Grid item xs={12}>
+                      <Typography variant="h6">Recently Found</Typography>
+                    </Grid>
+                  </Grid>
+                }
+              />
+            </Card>
+            {reportHeader('Recently Found')}
+            {foundReports.map((report) => reportRow(report, true))}
+            {/* Conditional message under the "Found" items */}
+            <Grid item xs={12} style={{ marginTop: '0.3rem' }}>
+              <Typography variant="inherit" align="center" color="var(--mui-palette-success-main)">
+                Check your email for an update on your item.
```

Comment:
Should probably say ```item(s)```

Reply:
sure. i feel its easier if its just item, because, i dont want it to show items for more than one and item for just one, that feels like a waste but then again. Ah well
I also dont want it to show items for just one item

Reply:
Well, the alternative is a little ternary (booleanValue ? somethingToDoWhenTruthy : somethineToDoWhenFalsy) logic to say ```(foundItems.length > 1 ? "items" : "item")```.  So either the parentheses s, or the conditional ```"s"``` is fine.